frame/staking/src/tests.rs

```diff
@@ -5726,6 +5726,46 @@ fn scale_validator_count_errors() {
 	})
 }
 
+#[test]
+fn set_min_commission_works_with_admin_origin() {
+	ExtBuilder::default().build_and_execute(|| {
+		// no minimum commission set initially
+		assert_eq!(MinCommission::<Test>::get(), Perbill::from_percent(0));
```

Comment:
nitpick, use Perbill::zero() ?

Reply:
or `Zero::zero()`, which will work if you move from `Perbill` to `Permill` or others :))